### PR TITLE
[expo-we-browser] Add secondaryToolbarColor to NCL example.

### PR DESCRIPTION
**Pull request by mczernek**
# Why

This option was added to WebBrowser recently in [this PR](https://github.com/expo/expo/pull/8615)

# Test Plan

Haven't found any browser that supports this option, can be tested by attaching debugger to WebBroserModule.

### CODE REVIEW

File: apps/native-component-list/src/screens/WebBrowserScreen.tsx [hunk 1]
```diff
@@ -176,6 +184,15 @@ export default class WebBrowserScreen extends React.Component<object, State> {
...
       <>
+        <View style={styles.label}>
+          <Text>Secondary toolbar color (#rrggbb):</Text>
+          <TextInput
+            style={styles.input}
+            placeholder="RRGGBB"
+            onChangeText={this.handleSecondaryToolbarColorInputChanged}
+            value={this.state.secondaryToolbarColor}
+          />
+        </View>
```
Comment (tsapeta) [inline apps/native-component-list/src/screens/WebBrowserScreen.tsx:195]: Shouldn't we render it only on Android? 🤔 Would be easier to test during QA 😉

**Review by tsapeta — APPROVED** [review]
Looks good 👍 
Maybe we could ask the author of this PR about the browser he uses?

File: apps/native-component-list/src/screens/WebBrowserScreen.tsx (hunk 1 above)
Comment (mczernek) [reply apps/native-component-list/src/screens/WebBrowserScreen.tsx:195]: It is only on android. 'renderAndroidChoices' is used only on this one platform :)

**Review by bbarthec — APPROVED** [review]

File: apps/native-component-list/src/screens/WebBrowserScreen.tsx (hunk 1 above)
Comment (tsapeta) [reply apps/native-component-list/src/screens/WebBrowserScreen.tsx:195]: Ah, right, I haven't looked above 🤦‍♂️

**Pull request merged**